### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by rocky1207**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import classes from './header.module.css';
```
Comment (kimnovak) [inline src/components/Header/Header.tsx:4]: Can you rename this to styles for consistency? `import styles from './header.module.css'`

File: src/components/Header/header.module.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        monospace;
+}
+header {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+        'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
+        'Helvetica Neue', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+
+    max-width: 1400px;
```
Comment (kimnovak) [inline src/components/Header/header.module.css]: Header should always take up the full width of the screen

File: src/components/Header/header.module.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+        monospace;
+}
+header {
```
Comment (kimnovak) [inline src/components/Header/header.module.css:4]: The header should be sticky, I don't see it in CSS, if it's not implemented please update it

File: src/components/Header/header.module.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        'Helvetica Neue', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    position: relative;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 19px 30px 11px 30px;
+    background-color: #27293d;
```
Comment (SreckoStojic) [inline src/components/Header/header.module.css]: You should import theme.module.css and use named colors in that Miljana created.

Comment (Zola987) [reply src/components/Header/header.module.css]: Yes, everything else is bellisimo.

**Review by bojanab987 — CHANGES_REQUESTED** [review]
Can you please merge Miljana's PR so that you can import theme.module.css in your stylesheet, and use variables she defined?

**Review by bojanab987 — APPROVED** [review]

**Pull request merged**